zerver/openapi/zulip.yaml

```diff
@@ -448,6 +448,206 @@ paths:
+                                "to": [4],
+                                "topic": "",
+                                "content": "What if we made it possible to sync drafts in Zulip?",
+                                "timestamp": 1595479020.43916,
+                              },
+                            "3":
+                              {
+                                "type": "private",
+                                "to": [4, 10],
+                                "topic": "",
+                                "content": "What if we made it possible to sync drafts in Zulip?",
+                                "timestamp": 1595479021.43916,
+                              },
+                          },
+                      }
+    post:
+      operationId: create_drafts
+      tags: ["drafts"]
+      description: |
+        Create one of more drafts for the current user.
+
+        `POST {{ api_url }}/v1/drafts`
+      parameters:
+        - name: drafts
+          in: query
+          description: |
+            A json-encoded list of all of the drafts that should be created for the current user.
+          schema:
+            type: array
+            items:
+              $ref: "#/components/schemas/Draft"
+      responses:
+        "200":
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccess"
+                  - properties:
+                      ids:
+                        type: array
+                        description: |
+                          An array of the IDS for the drafts that were just created in the same
+                          order as they were requested.
+                        items:
+                          type: integer
+                  - example: {"result": "success", "msg": "", "ids": [1, 2, 3]}
+        "400":
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/CodedError"
+                  - example:
+                      {
+                        "code": "BAD_REQUEST",
+                        "msg": "Must specify exactly 1 stream ID for stream messages",
+                        "result": "error",
+                      }
+  /drafts/{draft_id}:
+    patch:
+      operationId: edit_draft
+      tags: ["drafts"]
+      description: |
+        Edit a single draft.
+
+        `PATCH {{ api_url }}/v1/drafts/{draft_id}`
+      parameters:
+        - name: draft_id
+          in: path
+          schema:
+            type: integer
+          description: |
+            The ID of the draft you want to edit.
+          required: True
+        - name: draft
+          in: query
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Draft"
```

Comment:
Should this schema be modified to not have an `id` field in it?

Reply:
Sorry, I don't get what you're suggesting here. Could you please elaborate? The `draft_id` field is necessary in the schema since it's a parameter that's part of the path.

Reply:
Ahh, OK, I had been concerned that this meant it needed to be in the dictionary as well.